pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -879,7 +905,14 @@ func (oe *operationExecutor) generateUnmountDeviceFunc(
 				deviceToDetach.VolumeSpec.Name(),
 				err)
 		}
-
+		refs, err := attachableVolumePlugin.GetDeviceMountRefs(deviceMountPath)
```

Comment:
This may happen as follows
1. create a set of pods through replication controller
2. delete replication controller (and their associated pods) with grace-period=0 AND at the same time restart kubelet
3. after kubelet restarts, some of the pods (but not all of them) still in api server, kubelet will receive those pods information and add them to desired state. Then reconciler will add them to actual state too.  (Now actual and desired states have some pods volumes info, but not all of them)
4. Some pod DELETE requests come in, the reconciler will try to unmount volumes and then unmountdevice. This will cause that unmount device when not all of the volumes are unmounted yet